### PR TITLE
Automatic zip and release with Travis CI

### PR DESCRIPTION
**Pull request by jswetzen**
I was looking for a free way to use your collection and found [Power Music Reader](https://itunes.apple.com/se/app/power-music-reader/id523120705?mt=8). It accepts zip archives for import so I automated the zipping so it’s ready for download from github whenever a new song is added! You can see the releases it creates in [my fork](https://github.com/jswetzen/worship/releases).

If you want to use this, you need to connect github to Travis  if you haven’t already, and change the secure api_key to one you’ve generated, because the one in the file now is for my repo. I can provide more detailed instructions if needed.

### CODE REVIEW

**Comment by mattgraham** [comment]
This is a great idea; i'm going to look into what it would do to onsong to have the .travis file yanked down as well. Currently this repo publishes to S3 bucket on updates and @onsong downloads these files. I would love for their to be Git methods in onsong but, I've not had time to write something new myself so we'll work with what we have. 

Thanks for the PR!

File: Shekinah Glory.onsong [hunk 1]
```diff
@@ -3,18 +3,46 @@ Artist: Cory Asbury
...
-We want more, we want more, we want more, we want more
-We want more, we want more, more of Your Spirit
+Chorus 2:
+You m[C]ove and we want more.
+You sp[G]eak and we want more
+You m[Dm]ove and we want more,
+more of Your Sp[F]irit.
+
+Instrumental:
+C    G    Dm
```
Comment (mattgraham) [inline Shekinah Glory.onsong]: These chords won't transpose within the application without being in the [ ] 's. Having them is great but if you transpose the song up a key/two/etc. all the rest of the chords will change and they'll remain the same.

**Comment by jswetzen** [comment]
Ah, I didn't know you had a system in place. Maybe onsong can download zips as well and then you don't even need S3?
By the way, the Shekinah change should be in another PR and you should probably not even merge this since you need to change the key anyway.

**Comment by jswetzen** [comment]
Ah, I didn't know you had a system in place. Maybe onsong can download zips as well and then you don't even need S3?
By the way, the Shekinah change should be in another PR and you should probably not even merge this since you need to change the key anyway.

**Comment by jswetzen** [comment]
I'll close this for now. You know where to find travis.yml and I'll put the song in another PR.

**Pull request closed without merging**